src/main/webapp/app/home/home.component.ts

```diff
@@ -39,8 +40,11 @@ export class HomeComponent implements OnInit, AfterViewChecked {
     captchaRequired = false;
     credentials: Credentials;
     isRegistrationEnabled = false;
+    passwordLoginDisabled = false;
+    loginFormOverride = false;
+    saml2Config: Saml2Config | undefined;
```

Comment:
we wanted to keep the home component independent of Saml2Config.
Is it really needed?